docs/content/guides/dagster-pipes/integrating-databricks-with-dagster-pipes.mdx

```diff
@@ -1,6 +1,305 @@
 ---
 title: "Integrating Databricks with Dagster Pipes | Dagster Docs"
-description: ""
+description: "Learn to integrate Dagster Pipes with Databricks to launch external code from Dagster assets."
 ---
 
 # Integrating Databricks with Dagster Pipes
+
+In this guide, we’ll show you how to use [Dagster Pipes](/guides/dagster-pipes) with Dagster’s Databricks integration to launch Databricks jobs. You can then send information such as structured metadata and logging back to Dagster, where it will be visible in the Dagster UI.
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on using an out-of-the-box
+  Databricks resource. For further customization, use the{" "}
+  <a href="/guides/dagster-pipes/customizing-dagster-pipes-protocols">
+    <code>open_pipes_session</code> approach
+  </a>{" "}
+  instead.
+</Note>
+
+---
+
+## Prerequisites
+
+To use Dagster Pipes with Databricks, you’ll need:
+
+- **To have the following libraries installed**:
+
+  - **Dagster (`dagster`) and the Dagster UI (`dagster-webserver`).** Refer to the [Installation guide](/getting-started/install) for more info.
+  - **The `dagster-databricks` library**:
+
+    ```shell
+    pip install dagster-databricks
+    ```
+
+- **In Databricks:**
+  - **An existing Databricks workspace** with an existing project that’s deployed with a Databricks job. If you don’t have this, follow the [Databricks quickstart](https://docs.databricks.com/workflows/jobs/jobs-quickstart.html) to set one up.
+  - **The following information about your Databricks workspace**:
+    - `host` - The host address, starting with `https://`, of your Databricks workspace
+    - `token` - A personal access token for the Databricks workspace Refer to the Databricks API authentication documentation for more info about retrieving these values.
+  - **To have the `dagster-pipes` library available in the Databricks environment.**
+
+---
+
+## Step 1: Create a Databricks job in Dagster
+
+In this step, you’ll create a Dagster asset that, when materialized, opens a Dagster pipes session and launches a Databricks job that executes some external code.
+
+### Step 1.1: Define a Dagster asset
+
+1. Create a new file named `dagster_databricks_pipes.py` in your Dagster project.
+
+2. Copy and paste the following imports into the file:
+
+   ```python
+   # dagster_databricks_pipes.py
+
+   from dagster import AssetExecutionContext, Definitions, EnvVar, asset
+   from dagster_databricks import PipesDatabricksClient
+   from databricks.sdk.service import jobs
+   ```
+
+3. Next, you’ll define the asset. Copy and paste the following below the imports in `dagster_databricks_pipes.py`:
+
+   ```python
+   # dagster_databricks_pipes.py
+
+   @asset
+   def databricks_asset(context: AssetExecutionContext, pipes_databricks: PipesDatabricksClient):
+       task = jobs.SubmitTask.from_dict({
+           "new_cluster": { ... },
+           "libraries": [
+               # must include dagster-pipes
```

Comment:
"must include dagster-pipes" -> "Make the latest version of dagster-pipes on PyPI available in the task environment"

Reply:
Is there a way to specify the version here? Or would this automatically use the latest version?